### DISCUSSION TITLE
Specifying order of appearance of Markers in plotMarkerHeatmap()

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#1360] [General]
When querying for specific markers for use in  plotMarkerHeatmap() how may I control the order in which the genes are plotted. I have played with binaryClusterrows and clusterCols. That is not doing it for me.

Thanks!

**Comment by rcorces** [comment]
We dont have a way to provide this type of control within ArchR. You would have to control this through ComplexHeatmap.
https://jokergoo.github.io/ComplexHeatmap-reference/book/a-single-heatmap.html#row-and_column_orders

**Reply by sylestiel** [reply]
Thank you!
